mock-http-api.json

```diff
@@ -75,15 +75,15 @@
             },
             {
               "name": "FUNCTIONS_EXTENSION_VERSION",
-              "value": "~3"
+              "value": "~4"
             },
             {
               "name": "FUNCTIONS_WORKER_RUNTIME",
               "value": "node"
             },
             {
               "name": "WEBSITE_NODE_DEFAULT_VERSION",
-              "value": "~14"
+              "value": "~18"
```

Comment:
Can we updated to Node 20?

Reply:
It is still in public preview, https://azure.microsoft.com/en-us/updates/public-preview-azure-functions-support-for-nodejs-20/#:~:text=Published%20date%3A%20September%2027%2C%202023&text=You%20can%20now%20develop%20functions,your%20Function%20Apps%20to%20Node. So, do you still want to use node20?

Reply:
Another thing is, if we use node20, there is warning error from `@azure/msal-node@1.24.2` as

```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@azure/msal-node@1.14.2',
npm WARN EBADENGINE   required: { node: '10 || 12 || 14 || 16 || 18' },
npm WARN EBADENGINE   current: { node: 'v20.8.0', npm: '9.7.2' }
npm WARN EBADENGINE }
```

Reply:
Yes, because it is LTS in 12 days: https://github.com/nodejs/release#release-schedule

Reply:
To upgrade to Node v20, Azure function has to migrate to programming model v4 which has a big change. So, we will use Node v18 for now. It will be another PR to upgrade to Node v20.